plugins/tableselection/plugin.js

```diff
@@ -1122,12 +1181,28 @@
 			CKEDITOR.document.appendStyleSheet( this.path + 'styles/tableselection.css' );
 		},
 
+		isElementIgnored: function( target ) {
+			if ( !target ) {
+				return false;
+			}
+
+			for ( var i = 0; i < this._ignoredElements.length; i++ ) {
+				if ( this._ignoredElements[ i ].contains( target ) ) {
```

Comment:
This conditional does not work when table is compared to itself (as our implementation of `contains` does not allow passing the second parameter to the native `contains`). Probably it could be replaced with:

```suggestion
				if ( this._ignoredElements[ i ].contains( target ) || this._ignoredElements[ i ].equals( target ) ) {
```

Reply:
This function should also work with the container keeping a table, something like:
```html
<div id="tableContainer">